cli/setup-rack.sh

```diff
@@ -24,14 +24,16 @@ rack model import --clear ../RACK-Ontology/OwlModels/import.yaml
 rack model import ../Turnstile-Ontology/99-Utils/import.yaml
 rack model import ../GrammaTech-Ontology/import.yaml
 rack model import ../STR-Ontology/import.yaml
+rack model import ../DesCert-Ontology/import.yaml
+rack model import ../LM-Ontology/import.yaml
 
 rack nodegroups delete --yes --regexp --ignore-nonexistent "^ingest" "^query " "^Ingest-"
 rack nodegroups import ../Turnstile-Ontology/99-Utils/NodeGroups
 rack nodegroups import ../nodegroups/ingestion
 rack nodegroups import ../nodegroups/queries
 
-rack data import --clear ../RACK-Ontology/OwlModels/ARP-4754A.yaml
-rack data import --clear ../RACK-Ontology/OwlModels/DO-330.yaml
+# rack data import --clear ../RACK-Ontology/OwlModels/ARP-4754A.yaml
+# rack data import --clear ../RACK-Ontology/OwlModels/DO-330.yaml
 rack data import --clear ../RACK-Ontology/OwlModels/DO-178C.yaml
-rack data import --clear ../RACK-Ontology/OwlModels/MIL-STD-881D.yaml
-rack data import --clear ../Turnstile-Ontology/99-Utils/Data/Model.yaml
+# rack data import --clear ../RACK-Ontology/OwlModels/MIL-STD-881D.yaml
+rack data import ../Turnstile-Ontology/99-Utils/Data/Model.yaml
```

Comment:
The reason there was a `--clear` on each of these is that each loads into a separate data graph (and the clear only affects that one data graph). I think the clear flag should remain in place.

Reply:
Ah! Makes sense. I'll add --clear back in.

Reply:
Given that this isn't obvious, could you add a comment along the way telling people? Otherwise all these `--clear` are... clearly confusing :)